storage/src/mmr/operation.rs

```diff
@@ -48,20 +48,24 @@ impl<K: Array, V: Array> Operation<K, V> {
     const DELETE_CONTEXT: u8 = 0;
     const UPDATE_CONTEXT: u8 = 1;
 
-    /// Create a new operation from `key` and `op_type`.
-    pub fn new(key: K, op_type: Type<V>) -> Self {
+    /// Create a new update operation from `key` and `value`.
+    pub fn update(key: K, value: V) -> Self {
```

Comment:
nit: It feels weird to me to not accept `Type` in a single op as `new(t: Type)`.

I think we may want to do this given we return `to_type`?

Reply:
heh, that's how it was before but I thought this felt a little more convenient.  Added `new(t: Type<K,V>)` but left in these for convenience?